src/blog/2024/01/history-of-nodered.md

```diff
@@ -0,0 +1,46 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 
+authors: 
+image: 
+tags:
+    - posts
+    - flowfuse
+---
+
+
+<!--more-->
+
+In January 2013, I could have never foreseen that my fun little proof-of-concept project would become Node-RED, an open source low-code environment with millions of deployments in IoT and automation.
+
+Long before IoT became the ubiquitous term it is today, I was working in IBM’s Emerging Technology Group playing around with capturing data from devices and doing interesting things with it. The team focused on very fast-paced, short, proof-of-concept projects and was afforded time to learn new skills, innovate and work on side projects.
+
```

Comment:
@Hasmin-AC This will become a wall of text IMO, and hard to read. Do we need headings?

Reply:
@Hasmin-AC I'll give this a full review later today - there are a few tweaks I'd like to make to keep it more in my voice if that makes sense. I'll see what photos I can dig up that will be suitable.

Reply:
Done - my proposed changes have now been merged into this branch